gitpod-io/website
# redirect for `/docs/vscode-extensions`

fixes https://github.com/gitpod-io/gitpod/issues/14754

---

How to Test:

- Go to [`/docs/vscode-extensions`](https://deploy-preview-3040--gitpod-kumquat.netlify.app/docs/vscode-extensions) it should redirect to [`/docs/references/ides-and-editors/vscode-extensions`](https://deploy-preview-3040--gitpod-kumquat.netlify.app/docs/references/ides-and-editors/vscode-extensions)
